pkg/windows/windows.go

```diff
@@ -282,7 +282,7 @@ func (vm *windows) FileExists(path string) (bool, error) {
 
 func (vm *windows) Run(cmd string, psCmd bool) (string, error) {
 	if psCmd && !vm.defaultShellPowerShell {
-		cmd = remotePowerShellCmdPrefix + cmd
+		cmd = remotePowerShellCmdPrefix + cmd + "\""
```

Comment:
Are these changes compatible with other supported versions if we backport this?

Reply:
Yep, the e2e suite verifies this as we still run 2019 nodes on AWS + Azure.